content/en/security/application_security/setup/compatibility/nodejs.md

```diff
@@ -74,11 +74,12 @@ The following operating systems are officially supported by `dd-trace`. Any oper
 
 ##### App and API Protection Capability Notes
 
-Although Threat Protection is available for express >= 4 versions, the blocking of payloads on the body is only supported for applications using `body-parser` library.
+Although Threat Protection is available for express >= 4 versions, the blocking of payloads on the body is only supported for applications using either the [`body-parser`][45] or [`multer`][46] libraries.
 
 | Framework | Versions | Threat Detection supported? | Threat Protection supported? |
 |-----------|----------|-----------------------------|------------------------------|
 | express   | >=4      | {{< X >}}                   | {{< X >}}                    |
+| fastify   | >=2      | {{< X >}}                   | {{< X >}}                    |
 | nextjs    | >=11.1   | {{< X >}}                   |                              |
```

Comment:
Nit: could versions be formatted as in `App and API Protection Capability Notes` table ([this one](https://github.com/DataDog/documentation/pull/30224/files#diff-6eac0c38c7482fb1ac7397c062ec31f4b2ecfe5c490dbf9b804e0f4d2412c82fL126-L143))